### PR TITLE
Fucking fixing the graphs yo

### PR DESCRIPTION
**Pull request by thiderman**

### CODE REVIEW

**Comment by thiderman** [comment]
@mgoeppner Reviev ples

File: js/src/components/PostMatch.vue [hunk 1]
```diff
@@ -77,19 +77,47 @@ export default {
...
+      if (!match.isStarted) {
+        // If we're on the first match, there is no previous, so bail.
+        if (index === 0 && kind === 'tryouts') {
+          this.$set('tournament', Tournament.fromObject(tournament))
+          return
+        }
+
+        index = tournament.previous.index
+        kind = tournament.previous.kind
+
```
Comment (mgoeppner) [inline js/src/components/PostMatch.vue]: Such copy

File: js/src/components/PostMatch.vue [hunk 2]
```diff
@@ -77,19 +77,47 @@ export default {
...
+        if (kind === 'tryout') {
+          kind = 'tryouts'
+        } else if (kind === 'semi') {
```
Comment (mgoeppner) [inline js/src/components/PostMatch.vue]: Very paste

File: js/src/components/PostMatch.vue [hunk 3]
```diff
@@ -77,19 +77,47 @@ export default {
...
+          kind = 'semis'
+        }
+
+        if (kind === 'final') {
+          match = Match.fromObject(tournament[kind])
+        } else {
+          match = Match.fromObject(tournament[kind][index])
```
Comment (mgoeppner) [inline js/src/components/PostMatch.vue]: Wow

Comment (thiderman) [inline js/src/components/PostMatch.vue]: fak u

**Pull request merged**